### PR TITLE
adding HP points to Hero and Enemy and condition to live

### PR DESCRIPTION
**Pull request by TaySabrina**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (helladarion) [inline .idea/.gitignore:1]: We shouldn't have the dir .idea included in this review, it is just adding a bunch of random files not related to the code, please remove it.

Comment (TaySabrina) [reply .idea/.gitignore]: Done.

Comment (helladarion) [reply .idea/.gitignore]: Now I can't see your changes. I'll decline this Pull Request and then you start a new one.

Comment (helladarion) [reply .idea/.gitignore]: You have just created a new pull request, you should have fixed the current one.

**Pull request closed without merging**